input/postgres/server_stats.go

```diff
@@ -69,8 +69,23 @@ COALESCE((
 ), '0'::xid) as standby
 `
 
-func GetServerStats(ctx context.Context, logger *util.Logger, db *sql.DB, postgresVersion state.PostgresVersion, systemType string) (state.PostgresServerStats, error) {
+const ioStatisticSQLPg16 string = `
+SELECT backend_type,
+	   object,
+	   context,
+	   reads,
+	   writes,
+	   extends,
+	   op_bytes,
+	   evictions,
+	   reuses,
+	   fsyncs
+  FROM pg_stat_io
```

Comment:
@seanlinsley We should probably add the `_time` fields as well here - AFAIK they didn't exist yet when I first prototyped this, but were added later (and are very useful to understand whether its likely the OS actually issued an I/O request to disk, vs getting something from the page cache).

We could probably also add `writebacks` and `writeback_time` (https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-IO-VIEW) - its a bit specialized, but potentially useful.